cves/CVE-2017-16840.yml

```diff
@@ -146,10 +155,12 @@ unit_tested:
     In FFmpeg, they have the FATE suite of regression tests. They are not
     "unit" tests per se, but are automated regression tests. If the fix for the
     vulnerability involved adding or updating a FATE suite, then make fix: true.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: |
+    Looking through the FATE suite, there does not seem to be any tests
+    for this subsystem.
+  fix: false
+  fix_answer: As there are no tests, there are no tests to fix for the subsystem.
```

Comment:
http://coverage.ffmpeg.org/
I found the file fixed on this website. Please check here.